detekt-rules-style/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/MisusedAlsoSpec.kt

```diff
@@ -0,0 +1,92 @@
+package io.gitlab.arturbosch.detekt.rules.style
+
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.test.assertThat
+import io.gitlab.arturbosch.detekt.test.compileAndLint
+import org.junit.jupiter.api.Test
+
+class MisusedAlsoSpec {
+    val subject = MisusedAlso(Config.empty)
+
+    @Test
+    fun `does not report when no also is used`() {
+        val code = """
+            class Test {
+                private var a = 5
+                
+                init {
+                    a.let { 
+                        it.plus(5)
+                        it.minus(10)
+                    }
+                }
+            }
```

Comment:
you can remove most of the test code focusing on the expression only:
```suggestion
                fun f(a: Int) {
                    a.let { 
                        it.plus(5)
                        it.minus(10)
                    }
                }
```
this also makes the tests more realistic IMO.